client_api/data_access/gumbo_data_access.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+import pandas as pd
+
+
+class GumboDAO:
+    """Data Access Object for """
+    def __init__(
+        self,
+        psycopg2_connection,
+        sanity_check=True,
+        username=None,
```

Comment:
I think we should remove username from the instance of this object and take it from the request.